isis/src/control/objs/BundleSolutionInfo/BundleSolutionInfo.cpp

```diff
@@ -1009,6 +1042,43 @@ namespace Isis {
     int numUsed;
     int imageIndex = 0;
 
+    char imgMeasuresH1[] = "        Measures        ";
+    char imgMeasuresH1a[]= "************************";
+    char imgMeasuresH2[] = "|Accepted    |    Total|";
+    char imgMeasuresH3[] = "             RMS(pixels)             ";
+    char imgMeasuresH3a[] ="*************************************";
+    char imgMeasuresH4[] = "|Samples    |    Lines    |    Total|";
+
+
+
+    //int h1 = (int)(sizeof(imgMeasuresH1)/sizeof(char) );
+    int h2 = (int)(sizeof(imgMeasuresH2)/sizeof(char) );
+    //int h3 = (int)(sizeof(imgMeasuresH3)/sizeof(char) );
+    int h4 = (int)(sizeof(imgMeasuresH4)/sizeof(char) );
+
+
+
+    int firstColumnWidth
+        = m_statisticsResults->observations().at(0)->at(0)->fileName().toLatin1().length();
+    //Find the maximum file name length for the first image (length of first column)
+    for (int i = 1; i < numObservations;i++) {
+      int numImagesInObservation = m_statisticsResults->observations().at(i)->size();
+      for (int j = 1; j < numImagesInObservation; j++ ) {
+        int l = m_statisticsResults->observations().at(0)->at(0)->fileName().toLatin1().length();
```

Comment:
Maybe a more descriptive name for your int here?

Reply:
(It looked like the number one to me at first...)

Reply:
This code was deleted.  It was a block of code set to determine the maximal file name size in the observation list, but it's ramifications ended up being more trouble than they were worth.  So I deleted and refactored that section of code.